schema/spacewalk/postgres/triggers/suseImageInfo.sql

```diff
@@ -27,6 +27,7 @@ execute procedure suse_imginfo_mod_trig_fun();
 CREATE OR REPLACE FUNCTION suse_image_info_image_removed_trig_fun() RETURNS TRIGGER AS $$
 BEGIN
   DELETE FROM suseSaltPillar WHERE id = OLD.pillar_id;
+  DELETE FROM suseSaltPillar WHERE category = concat('SyncedImage', OLD.id);
```

Comment:
How/when are we going to remove "LegacySyncedImage" pillars? Only when they are removed from RBSs?

Reply:
Yes, they are removed when RBS sends the "remove" event.